@here/harp-examples/webpack.config.js

```diff
@@ -123,31 +103,32 @@ function filterExamples(pattern) {
 //
 //filterExamples("hello");
 
-const browserConfig = merge(commonConfig, {
-    entry: webpackEntries,
-    output: {
-        filename: "[name]_bundle.js"
-    },
-    optimization: {
-        splitChunks: {
-            chunks: "all",
-            minSize: 1000,
-            name: "common"
+const browserConfig = addHarpWebpackConfig(
+    merge(commonConfig, {
+        optimization: {
+            splitChunks: {
+                chunks: "all",
+                minSize: 1000,
+                name: "common"
+            }
         }
+    }),
+    {
+        mainEntry: webpackEntries
     }
-});
+)[0];
 
-const exampleBrowserConfig = merge(commonConfig, {
-    entry: {
+const exampleBrowserConfig = addHarpWebpackConfig(commonConfig, {
+    mainEntry: {
         "example-browser": "./example-browser.ts"
     }
-});
+})[0];
 
-const codeBrowserConfig = merge(commonConfig, {
-    entry: {
+const codeBrowserConfig = addHarpWebpackConfig(commonConfig, {
```

Comment:
Now that's total misuse.
This entry has nothing to do with `harp` and it created two configs only for second to be abandoned.

I think it shows, that we should have "two" entries, or option to configure if decoder should be created.

It adds CopyWebpackPlugin for`resources` which is completly unnecessary!

Possible examples:

```
const codeBrowserConfig = addHarpWebpackConfig(commonConfig, {
    noDecoder: true,
    noResources: ???,
    noWhatever: ...
})```

Or just expose "harpSourceCommon" config (with typescript, resolve options etc) without "application" level extras like decoder and default resources.